org.eclipse.xtext.util/src/org/eclipse/xtext/util/OnChangeEvictingCache.java

```diff
@@ -103,20 +103,27 @@ protected void cacheHit(CacheAdapter adapter) {
 	}
 	
 	/**
-	 * Returns the cache adapter that is associated with the resource. The lifecycle of the cache
+	 * Returns the cache adapter that is associated with the resource. The life cycle of the cache
```

Comment:
looks like we are all but consistent on how to write life cycle

Reply:
As usual 
I recently enabled spell checking in my workspaces. Now I'm trying to be a good boy scout when I come across "unconventional" spelling.